schedule/lvm_thin_provisioning.yml

```diff
@@ -0,0 +1,77 @@
+name:           lvm_thin_provisioning
+description:    >
+    Complete OS deployment with unencrypted LVM drive management.
+    Test creates 2 LVM and BIOS boot partitions.
+    Thin pool and thin lv resides on the second LVM partition,
+    where /home (XFS) is being mounted. Maintainer: mloviska, poo#39023
```

Comment:
There is opportunity here to drag the maintainer (if not strong opposition)

Reply:
It's PoC still and won't be merged as it is. As of now name and description fields are not used anywhere and I've copied it 1 to 1, so if one decides to remove maintainer, we should already have done it =).